index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>HTML Mastery Quiz</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<div id="wrapper">
+		<article id="card">
+			<div class="section1">
```

Comment:
It's a good idea to avoid naming classes with order or count numbers, because it ties you to a single layout. If you needed to add an additional card or section between two existing ones, it would throw off your class names, most likely affecting your CSS, and generally making it harder to maintain the code.